Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -49,7 +52,7 @@ def __init__(self, *args, **kwargs):
         #self._PCI_ENUM_BUS_SCAN_TYPE    = 1
         self._PCI_ENUM_BUS_SCAN_ITEMS = '0,10,0x35,128,0xff'
         self.PCI_IO_BASE              = 0x00002000
-        self.PCI_MEM32_BASE           = 0x80000000
+        self.PCI_MEM32_BASE           = 0x80000000 if self.ENABLE_SIMICS != 1 else '0xF0000000'

```

Comment:
This need to be updated at runtime through patchable PCD.

Reply:
Done.